11-dynamic-inventory/README.md

```diff
@@ -52,21 +50,19 @@ Modify the `local.docker.yml` to lookup the environment variables for all the co
 +    key: 'docker_config.Env'
 ```
 
-Now re-run the command to list the groups and see if there's a group that would only include the `ansible_katas_host_1` and `ansible_katas_host_2` containers:
+4. Now re-run the command to list the groups and see if there's a group that would only include the `ansible_katas_host_1` and `ansible_katas_host_2` containers:
 
 ```bash
-ansible-inventory -i local-docker.yml --graph
+ansible-inventory -i local.docker.yml --graph
 ```
 
-In reality there might be many matches here, but we know we're interested in the `GROUP` environment variable and it's value. So next we can use that group in a playbook, just like in the example playbook:
-
-```bash
-cat playbook.yml
-```
+Look for the `GROUP` environment variable and it's value, showing up as `env_GROUP_hosts`. Next we can use that group in a playbook, just like in the example playbook `playbook.yml`
 
 ### Executing Playbooks with Dynamic Inventory
 
-You might be wondering how to put this all together by now, so let's finally execute this playbook by utilising the dynamic inventory:
+Now, let's finally execute a test playbook by utilising the dynamic inventory.
+
+5. Run the command below:
```

Comment:
Should we first examine the `playbook.yml` with `cat` or something? OTOH it could be left "as exercise for the reader".

Reply:
I thought about this as well but since the test playbook was very self explanatory and simple in its output I left it out to keep things a bit more condensed, not sure what is best